frontend/class-opengraph-image.php

```diff
@@ -92,6 +99,19 @@ public function __construct( $image = null, WPSEO_OpenGraph $opengraph = null )
 		}
 	}
 
+	/**
+	 * Gets the class for determine the current page type.
+	 *
+	 * @return WPSEO_Frontend_Page_Type
+	 */
+	private function get_frontend_page_type() {
```

Comment:
I would make this `protected` because it contains a dependency and by making it protected you can mock this method for unittests.